.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: "gitsubmodule"
+    directory: "/"
+    allow:
+      - dependency-name: "moose"
```

Comment:
Which moose branch does it pull from? next or devel or master? Their "main" branch is next, but I think devel would be safer from potential bugs

Reply:
It pulls from the default branch, I am now realizing the default is the "next" branch. I will go ahead and try to fix this!

Reply:
I have changed the target branch for the MOOSE submodule to the devel branch.